app/src/main/kotlin/com/intive/picover/profile/viewmodel/ProfileViewModel.kt

```diff
@@ -73,15 +71,20 @@ class ProfileViewModel @Inject constructor(
 		}
 	}
 
-	fun validatingName(text: String) =
-		textValidator.validate(text)
+	fun onUsernameChange(username: String) {
+		_username.value = username
+	}
+
+	private fun validateUsername(text: String) =
+		textValidator.validate(text).errorMessageId
```

Comment:
It can be inlined, will be easier to read what's going on in `usernameErrorMessageId`

Reply:
Done

Reply:
I meant to get rid of `validateUsername` function and put its body to line 33

Reply:
Done